cliff.toml

```diff
@@ -68,6 +68,17 @@ filter_unconventional = true
 split_commits = false
 # regex for preprocessing the commit messages
 commit_preprocessors = [
+  # Fixes for specific commit messages that we wish we'd committed differently
+  { pattern = """^bump tonic\\/prost dependencies and MSRV to 1.64 \\(#464\\)\\s+
+(.+\\s+.+)""", replace = """feat(api): update `tonic` to v0.10 and increase MSRV to 1.64 (#464)
+
+BREAKING CHANGE:
+This is a breaking change for users of `console-api` and `console-subscriber`,
+as it changes the public `tonic` dependency to a semver-incompatible version.
+This breaks compatibility with `tonic` 0.9. and `prost` 0.11.""" },
```

Comment:
sigh...this is a good call, but it's unfortunate that we have to do this. maybe at some point we should go back and edit the git history so that that commit isn't screwy...this looks good for now though.

Reply:
Removed now that you updated the commit.